package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "zen-audio-player-electron",
+  "version": "0.1.0",
+  "main": "main.js",
+  "dependencies": {
+    "electron-prebuilt": "^1.2.7"
+  },
+  "devDependencies": {
+    "eslint": "^3.1.1",
+    "eslint-config-google": "^0.6.0",
```

Comment:
I think the reason for the "eslint-config-google" is that I originally indicated the google standard for linting. Then, I just copied your .eslint config file which had custom settings. I can remove it and see what happens.